### ISSUE TITLE
addressPostalCode from Billing Address is not saved on strapi

### ISSUE THREAD

**Issue by simona-iuliana** [stripe/stripe-react-native#613]
I'm trying to send the postalCode in billingAddress along with country, the country is saved in Strapi, but the postal code isn't.  
The weird thing is that this works on ios, but not on android. 

On the CardField I used  `postalCodeEnabled={false}` and the postal code is on a separate textinput and i send it like that:

` const billingDetails: PaymentMethodCreateParams.BillingDetails = {
      addressCountry: values.countryCode || CountryCode.US,
      addressPostalCode: values.addressZipCode,
    };`
 On IOS works, the postal code is saved in Strapi on that billing address field, but on Android isn't.

 - Device: Android

i'm using "@stripe/stripe-react-native": "0.2.0"

**Comment by simona-iuliana** [comment]
I solve this issue with patch-package. In android/src/main/java/com/reactnativestripesdk/Mappers.kt at line 433. The problem was that the postalcode from cardAddres isn't null, is an empty string and that overwrite the postalCode from the billing address. 
This is what I did, I check for empty strings too:

  cardAddress?.let { ca ->
      ca.postalCode?.let {
        if (ca.postalCode != "") {
          address.setPostalCode(it)
        }
      }
      ca.country?.let {
        if (ca.country != "") {
          address.setCountry(it)
        }
      }
    }

**Comment by acomley-stripe** [comment]
Correct. If this is also an issue on iOS I will re-open and prioritize.
